include/xsimd/arch/xsimd_wasm.hpp

```diff
@@ -385,6 +385,18 @@ namespace xsimd
             return wasm_f64x2_gt(self, other);
         }
 
+        // isnan
+        template <class A>
+        inline batch_bool<float, A> isnan(batch<float, A> const& self, requires_arch<wasm>) noexcept
+        {
+            return wasm_v128_or(wasm_f32x4_ne(self, self), wasm_f32x4_ne(self, self));
```

Comment:
This is I think so the first pure emulation that I've implemented, no direct intrinsic exists for any of the data types (float or double). I think the implementation should work
1) First I negate the equality between two batches ( the NAN elements would give 1 after `ne`, others would give 0)
2) Then we can use the bitwise_or operator to generates 1 (true) and 0 (false) elements.

Let me know if this is the correct way to handle this.

Reply:
LGTM